Products/ZenUI3/browser/search/resources/zenoss-search.js

```diff
@@ -59,18 +48,50 @@ Ext.onReady(function () {
             idProperty:'url',
             fields:[
                 {name:'url'},
```

Comment:
add "id","name" and "isSavedResult" fields to "live search" store to also handle in it records from "saved result" store and use one store and one picker in combo.